apps/frontend/src/pages/projects/libs/enums/color-class.enum.ts

```diff
@@ -0,0 +1,7 @@
+const ColorClass = {
+	GREEN: "green",
+	RED: "red",
+	YELLOW: "yellow",
+} as const;
```

Comment:
not sure about naming

Reply:
What about `LastActivityIndicatorColor`?

Reply:
Maybe it'd be better to name this enum as just Color, or ColorName?

Reply:
Oh, I see, it's for class names, now I'm not sure too, but think that ColorClassName would be pretty self-explanatory